README.md

```diff
@@ -6,16 +6,18 @@ ARK-OS is a collection of software services and tools for drones. These services
 - **ARK Pi6X Flow** <br> https://arkelectron.com/product/ark-pi6x-flow/
 
 # Getting started
+If there is no Wifi connection configured a hotspot should come up.
```

Comment:
Only after ARK OS is installed. I don't think this information should be here at the top of the README.